models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -536,6 +564,15 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
       if (!is.na(lai) && is.numeric(lai)) {
         param[which(param[, 1] == "laiInit"), 2] <- lai
       }
+
+      #Initial LAI is set as 0 for deciduous forests and grasslands for non-growing seasons
+      if (!(lubridate::month(settings$run$start.date) %in% seq(5,9))){ #Growing seasons are coarsely defined as months from May to September for non-conifers in the US
+         site_pft <- utils::read.csv(settings$run$inputs$pft.site$path)
+         site.pft.name <- site_pft$pft[site_pft$site == settings$run$site$id]
+         if (site.pft.name!="boreal.coniferous") {   #Currently only excluding boreal conifers. Other evergreen PFTs could be added here later.
```

Comment:
Not going to hold up this PR, but this chunk of code here needs updating before it's used more broadly. Even for North American runs, there's has to be a better way of knowing the phenology of a PFT than hard coding it (e.g. adding to the `<model>` or `<pft>` configs. Similarly, why hard code the LAI initialization uniformly based on months if we've got a model file that prescribes the leaf-on and leaf-off dates?